src/build_blurhash.py

```diff
@@ -5,23 +5,43 @@
 
 
 ffibuilder = cffi.FFI()
-ffibuilder.set_source('blurhash._encode', '''
-    const char* blurHashForPixels(int xComponents, int yComponents, int width,
-                                  int height, uint8_t *rgb,
-                                  size_t bytesPerRow);
+ffibuilder.set_source('blurhash._functions', '''
```

Comment:
`_functions` might be a bit too generic name, but I'll have to think a bit to come up with a good alternative. It is fine for now since renaming it later is not an issue.

Reply:
Yeah! I couldn't come up with a proper name for this, so leaving this as `_functions` as of now.